REVIEW.md

```diff
@@ -0,0 +1,209 @@
+
+- [ ] Motivation
+    - Do we want this?
+    - Would it benefit a meaningful number of users to have this accepted into Spyder?
+    - Does it help to solve bugs?
+    - Is it a necessary refactoring?
+
+- [ ] Tests
+    - Do the tests pass in the continuous integration build?
+      If appropriate, help the contributor understand why tests failed.
+    - Are regression tests needed?
+      If a reproducible bug is being fixed in this PR, one may be needed, and reviewers should help the contributor design and implement them.
+      In case the contributor is unable to add the test, the reviewer should do so in a follow-up PR.
+
+ - [ ] Sustainability and maintenance
+    - Is the code easy to read and low on redundancy?
+    - Should variable names be improved for clarity or consistency?
+    - Should comments or docstrings be added or removed?
+    - Will the cost of maintaining a new feature be worth its benefits?
+    - Will the new code add any dependencies on other projects?
+
+- [ ] User-facing changes
+    - What use cases might this change influence?
+    - Is in-interface text (new or modified) appropriately descriptive and free of spelling or grammar issues that would make its meaning unclear?
+
+#### Optional items
+
+The following are themes that may be helpful to cover on a PR, but are optional.
+
+**Note**: The difference between an objective improvement and a subjective preference isn’t always clear.
+Reviewers should recall that the review process is primarily about reducing risk to the project.
+When reviewing code, one should aim at preventing situations which may introduce a bug, deprecation or regression.
+
+- [ ] Improvements
+    - Could the code easily be rewritten to run much more efficiently for relevant settings?
+    - Is the code backwards compatible with previous versions?
+      (Or is a deprecation cycle necessary?)
+    - Is there feedback that would be helpful for solving future issues surrounding this change?
+      Are there non-urgent concerns that the team should be made aware of?
+    - If a bug is fixed, does the PR include a non-regression test?
+    - Are there images that should be added, resized or optimized?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns in Spyder?
+      If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Reviews may be tagged per comment or through the primary comment of a GitHub review.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+#### Discussion items
+
+I am not sure what people want for the discussion items tag, so this is a placeholder to make space for collaboration.
```

Comment:
@ccordoba12 @CAM-Gerlach You were the main two people that I think requested this additional tag (based on my notes). I'm not sure what you think fits here, so I'd like your input if possible.

Reply:
Commenting just to get some attention on this again. I think this section being empty is the only thing stopping this from being voted on. (Unless I'm misunderstanding.)

Reply:
One more comment for notifications and then I'll stop. I think it would be great to get this done and vote (as mentioned below) so we know what actions we're taking in the new year. Thanks!

Reply:
My suggestion for this section

> Reviewers can leave comments marked with the `[discussion]` tag (preferably), or not marked with any tag, to ask for clarifications from the contributor, suggest that some details of the implementation could be improved in the future, indicate that a test would be nice to have, or any other hint that could help the contributor or other reviewers to better understand the context of the proposed work. Contributors are not required to answer or address those comments as part of the review process.

Reply:
@CAM-Gerlach, what are you thoughts about this?